### ISSUE TITLE
Is there plan to release a cased TLM model?

### ISSUE THREAD

**Issue by yilinyang7** [facebookresearch/XLM#60]
Like one trained on cased En-De with MLM+TLM. I think it could help supervised MT.

Thanks.

**Comment by glample** [comment]
Hi,

Yes it would probably help, and we haven't tried. We don't have plans to do this any time soon, but what you can do is fine-tune the MLM De-En only model. Training MLM is slow, but once you have trained a MLM model, fine-tuning on MLM + TLM should be pretty fast and you don't need 64 GPUs to do that. This is what we did for XNLI: we trained a MLM only model, then fine-tuned it on MLM + TLM.

**Comment by yilinyang7** [comment]
For fine-tune training on MLM+TLM, did you also reduce the learning rate? By which scales then?

Thanks!

**Comment by glample** [comment]
Sorry for the delay. No, we used the same learning rate of 1e-4
